packages/html/src/render.ts

```diff
@@ -370,14 +405,17 @@ export function serializableEvent<T>(event: Event): T {
   for (const property of allowListedEventTargetProperties) {
     targetObject[property] = event.target?.[property as keyof EventTarget];
   }
-  if (
-    event.target instanceof HTMLSelectElement && event.target.selectedOptions
```

Comment:
We do not have `HTMLSelectElement` as a runtime type in Deno -- instead, duck type